### PR TITLE
Moves upgrade data and schema to install data and schema

### PR DESCRIPTION
**Pull request by netorodrigues**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| **Issue**           | https://mundipagg.atlassian.net/browse/PAOP-90
| **What?**         | Removes old mundipagg's databases changes based on versoning.
| **Why?**          | To start a new versioning way in pagarme's module.
| **How?**          | Moving all script changes made in upgrade scripts (data and schema) to install scripts (data and schema)

**Important Notes**
- this PR also removes methods that apply changes to tables that do not appear to be in use today by the module. These methods are all listed in **Recurrence.php** and **UpgradeData.php** files changed in this PR.
- This PR also solves a bug in magento's admin configuration page, datailed in https://mundipagg.atlassian.net/browse/PAOP-150 

#### :speech_balloon: Important guidelines

* Add pull request labels.
* Check base branch.
* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!

### CODE REVIEW

**Review by vberardi — DISMISSED** [review]

**Review by willian-soaresferreira — CHANGES_REQUESTED** [review]
Boa! Só deixei 2 detalhes mesmo como sugestão.

File: Setup/InstallData.php [hunk 1]
```diff
@@ -10,14 +10,20 @@
 class InstallData implements InstallDataInterface
 {
 
+    private $customerSetupFactory;
+    public function __construct(
+        \Magento\Customer\Setup\CustomerSetupFactory $customerSetupFactory
```
[the remark this answers is not in the document]
Comment (netorodrigues) [reply Setup/InstallData.php]: Fica melhor mesmo. Modificado!

File: Setup/InstallData.php [hunk 2]
```diff
@@ -28,6 +34,40 @@ public function install(
             $connection->update($tableName, $data, $where);
         }
 
+        $this->addsCustomerIdPagarme($setup);
```
[the remark this answers is not in the document]
Comment (netorodrigues) [reply Setup/InstallData.php]: Feito

**Comment by netorodrigues** [comment]
This PR was closed due to a rebase issue in develop branch. We needed to recreate it in order to get the correct changes, so we made it in #23

**Pull request closed without merging**